tempodb/config.go

```diff
@@ -60,8 +63,33 @@ type Config struct {
 }
 
 type SearchConfig struct {
+	// v2 blocks
 	ChunkSizeBytes     uint32 `yaml:"chunk_size_bytes"`
 	PrefetchTraceCount int    `yaml:"prefetch_trace_count"`
```

Comment:
`PrefetchTraceCount` is used in `vparquet` as well

Reply:
Seems like it's only used during compaction which defaults to 1000 and does not seem configurable.

Reply:
I'm not sure I understand what you mean - what is not configurable?

Reply:
I could be wrong, but the only place I can find us using a "PrefetchTraceCount" in parquet was during compaction. And that is set here and I don't think there's anyway to configure that using yaml.

https://github.com/grafana/tempo/blob/a7a308550b149835cd3dc206c8e00b7bce1d4a0e/tempodb/encoding/common/interfaces.go#L57